Bio/Sequencing/Applications/_samtools.py

```diff
@@ -321,47 +323,95 @@ class SamtoolsSortCommandline(AbstractCommandline):
     Concatenate BAMs, equivalent to::
 
     $ samtools sort [-no] [-m maxMem] <in.bam> <out.prefix>
+    $ samtools sort [-n] [-T FREFIX] [-o file] [-I INT] [-m maxMem] <in.bam>
 
     See http://samtools.sourceforge.net/samtools.shtml for more details
 
     Example
     -------
 
     >>> from Bio.Sequencing.Applications import SamtoolsSortCommandline
-    >>> input_bam = "/path/to/input_bam"
-    >>> out_prefix = "/path/to/out_prefix"
-    >>> samtools_sort_cmd = SamtoolsSortCommandline(\
-                                                    input_bam=input_bam,\
-                                                    out_prefix=out_prefix)
-    >>> print(samtools_sort_cmd)
-    samtools sort /path/to/input_bam /path/to/out_prefix
+    >>> import subprocess
+    >>> child = subprocess.Popen(\
+                str("samtools"), \
+                stdout=subprocess.PIPE, \
+                stderr=subprocess.PIPE, \
+                universal_newlines=True)
+    >>> stdoutdata, stderrdata = child.communicate()
```

Comment:
This has effectively made the ``samtools`` binary a dependency for running the Biopython doctests - something we've avoided until now.